sdk/containerinstance/Microsoft.Azure.Management.ContainerInstance/src/Generated/Models/ContainerGroup.cs

```diff
@@ -21,7 +21,7 @@ namespace Microsoft.Azure.Management.ContainerInstance.Models
     /// A container group.
     /// </summary>
     [Rest.Serialization.JsonTransformation]
-    public partial class ContainerGroup : Resource
+    public partial class ContainerGroup
```

Comment:
Is this expected?

Reply:
This was autogenerated from new swagger spec. The build and tests passed, don't see any issues with it.